### PR TITLE
feat: Add small styling changes

### PR DESCRIPTION
**Pull request by amfestoy**

### CODE REVIEW

File: src/stories/Forms/Checkbox/index.tsx [hunk 1]
```diff
@@ -23,10 +23,6 @@ const FakeCheckbox = styled.label<{ hasError: boolean }>`
     border-radius: ${rem(2)};
     content: '';
   }
```
Comment (RostiMelk) [inline src/stories/Forms/Checkbox/index.tsx:25]: Why is this removed?

File: src/stories/Forms/InputWrapper/index.tsx [hunk 2]
```diff
@@ -26,7 +26,8 @@ const Wrapper = styled.div<WrapperProps>`
     background-color: var(--color-alertRedBg);`}
 
   // apply left margin for certain input types
-  label {
```
Comment (RostiMelk) [inline src/stories/Forms/InputWrapper/index.tsx:29]: 🙌🙌🙌🙌🙌

File: src/stories/Forms/Select/Option.tsx [hunk 3]
```diff
@@ -32,7 +32,6 @@ const Wrapper = styled.li`
 `;
 
 const StyledCheckbox = styled(Checkbox)`
-  margin-left: calc(var(--spacing-4) * -1);
```
Comment (RostiMelk) [inline src/stories/Forms/Select/Option.tsx:35]: This still needs to be here

File: src/stories/Forms/Checkbox/index.tsx (hunk 1 above)
Comment (amfestoy) [reply src/stories/Forms/Checkbox/index.tsx:25]: Because it is added to global styles

File: src/stories/Forms/InputWrapper/index.tsx [hunk 4]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import styled from 'styled-components';
 import rem from '../../../shared/pxToRem';
 import { AlertText, Checkbox, Radio, Toggle } from '../../../index';
+import { ErrorSeverity } from 'src/shared/types';
```
Comment (RostiMelk) [inline src/stories/Forms/InputWrapper/index.tsx:5]: bad import

File: src/stories/Forms/InputWrapper/index.tsx [hunk 5]
```diff
@@ -132,23 +129,10 @@ export const InputWrapper = React.forwardRef(
...
-        {hint && !alert && (
-          <HintWrapper {...HintWrapperProps}>
-            <HintText id={ariaErrorMessageId} className='hint-text'>
-              {hint}
-            </HintText>
-          </HintWrapper>
-        )}
-
-        {alert && (
+        {severity && (
```
Comment (RostiMelk) [inline src/stories/Forms/InputWrapper/index.tsx:132]: Maybe also check for `severityMessage` prop, or add default messages for the different severity states?

File: src/stories/Forms/Label/index.tsx [hunk 6]
```diff
@@ -59,7 +55,7 @@ export const Label = ({
     <Wrapper className={className}>
       <StyledLabel htmlFor={htmlFor || id}>
         {children}{' '}
-        {!isRequired && (
+        {isRequired === false && (
```
Comment (RostiMelk) [inline src/stories/Forms/Label/index.tsx:58]: Awesome! This will be great

File: src/stories/Forms/Radio/Radio.stories.tsx [hunk 7]
```diff
@@ -40,11 +45,27 @@ InWrapper.args = {
...
 export const HasError = Template.bind({});
 HasError.args = {
   ...Primary.args,
-  alert: 'This is an error message.',
+  severity: 'error',
+  severityMessage: 'This is an error message.',
+};
+
+export const HasSuccess = Template.bind({});
+HasSuccess.args = {
```
Comment (RostiMelk) [inline src/stories/Forms/Radio/Radio.stories.tsx:67]: I really hope you automated all of these changes 😅

File: src/stories/Forms/Select/Option.tsx [hunk 8]
```diff
@@ -7,7 +7,8 @@ const Wrapper = styled.li`
   display: flex;
   position: relative;
   align-items: center;
-  padding: ${rem(13)} var(--spacing-16);
+  height: var(--spacing-48);
+  padding: 0 var(--spacing-16);
```
Comment (RostiMelk) [inline src/stories/Forms/Select/Option.tsx:11]: As discussed yesterday, I think we should continue to use margins here to avoid breaking UI when using long option labels.

File: src/stories/Foundation/AlertText/index.tsx [hunk 9]
```diff
@@ -23,21 +26,14 @@ export interface AlertTextProps {
...
-    case 'success':
-      return <InfoSmall />;
-    case 'info':
-      return <InfoSmall />;
-    case 'warning':
-      return <WarningSmall />;
-    case 'error':
-      return <WarningSmall />;
-  }
+const SEVERITY_STYLING: { [key: string]: { color: Colors; icon: JSX.Element } } = {
```
Comment (RostiMelk) [inline src/stories/Foundation/AlertText/index.tsx:29]: ✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨

**Review by RostiMelk — COMMENTED** [review]
Looks good! Just left some minor questions and comments.
